### ISSUE TITLE
Stick all of the modules together to create the final program

### ISSUE THREAD

**Issue by FebruaryRain** [FebruaryRain/PostcodeCrimeChecker#33]
`As the user, when I use the program I want it to all function together, so that I can get the end product (the report)`

As it stands, we have disparate modules which are not joined up.
Work needs to happen to organise this into a coherent structure.
In main.py:
 - [x] Organise run() so that it calls the UI.
 - [ ] Organise the central postcode lookup. (Not done, different check used)
 - [x] Organise the data sorting and filtering so that we get the final product.
 - [x] Organise the file writing so that we save the final product
Once this is done, we can start looking at adding further functionality.

**Comment by FebruaryRain** [comment]
Organise the central postcode lookup. (Not done, different check used):
James has already implemented a check to ensure that a postcode starts with EX. 
This is a separate issue which can be implemented if there is time.
